eng/Npm.Workspace.proj

```diff
@@ -0,0 +1,30 @@
+<Project>
```

Comment:
I think this should be more prominent that something inside targets, otherwise it's hard to find if we need to make adjustments. I would even go as far as to put it in the repo root.

I don't feel strongly about it, but we can always change it later as a follow up. Does that work?